docs/querying/sql-data-types.md

```diff
@@ -39,13 +38,12 @@ Druid natively supports the following basic column types:
 * COMPLEX: non-standard data types, such as nested JSON, hyperUnique and approxHistogram, and DataSketches
 * ARRAY: arrays composed of any of these types
 
-Timestamps (including the `__time` column) are treated by Druid as longs, with the value being the number of
+Druid treats timestamps (including the `__time` column) as LONG, with the value being the number of
 milliseconds since 1970-01-01 00:00:00 UTC, not counting leap seconds. Therefore, timestamps in Druid do not carry any
-timezone information, but only carry information about the exact moment in time they represent. See the
-[Time functions](sql-scalar.md#date-and-time-functions) section for more information about timestamp handling.
+timezone information, but only carry information about the exact moment in time they represent. See
+[Time functions](sql-scalar.md#date-and-time-functions) for more information about timestamp handling.
 
-Casts between two SQL types with the same Druid runtime type (see below table) have no effect, other than exceptions
-noted in the table. Casts between two SQL types that have different Druid runtime types generate a runtime cast in
+Casts between two SQL types with the same Druid runtime type have no effect, other than the exceptions noted in the following table. Casts between two SQL types that have different Druid runtime types generate a runtime cast in
```

Comment:
No comma between `effect` and `other`. The sequencing of this section is a little odd. The table comes 2 sentences after 2 other cases.

Maybe

Casts w/ same runtime
TABLE

Casts w/ different runtime

Can't cast